core/src/main/java/com/sequenceiq/distrox/v1/distrox/converter/InstanceTemplateParameterConverter.java

```diff
@@ -56,7 +56,7 @@ private AwsInstanceTemplateV4SpotParameters convert(AwsInstanceTemplateV1SpotPar
     private AwsEncryptionV4Parameters convert(AwsEncryptionV1Parameters source, DetailedEnvironmentResponse environment) {
         AwsEncryptionV4Parameters response = new AwsEncryptionV4Parameters();
         EncryptionType dataHubEncryptionKeyType = source.getType();
-        if (dataHubEncryptionKeyType.equals(EncryptionType.CUSTOM)) {
+        if (dataHubEncryptionKeyType != null && dataHubEncryptionKeyType.equals(EncryptionType.CUSTOM)) {
```

Comment:
No, the usual way to do this is swapping the objects around `equals()`. So
```
EncryptionType.CUSTOM.equals(dataHubEncryptionKeyType)
```
That construct will never result in an NPE.